.gitattributes

```diff
@@ -1,29 +1,6 @@
 # Auto detect text files and perform LF normalization
 * text=auto
 
-# Custom for Visual Studio
-*.cs     diff=csharp
-*.sln    merge=union
-*.csproj merge=union
-*.vbproj merge=union
-*.fsproj merge=union
-*.dbproj merge=union
-
-# Standard to msysgit
-*.doc	 diff=astextplain
-*.DOC	 diff=astextplain
-*.docx diff=astextplain
-*.DOCX diff=astextplain
-*.dot  diff=astextplain
-*.DOT  diff=astextplain
-*.pdf  diff=astextplain
-*.PDF	 diff=astextplain
-*.rtf	 diff=astextplain
-*.RTF	 diff=astextplain
-
 # Release export attributes
 technic/spec/** export-ignore
```

Comment:
While cleaning stuff I think this could also be wider match:
`**/spec/** export-ignore` or `**/spec export-ignore`

Here I think it might be good to "block" spec as directory name as it is convention for .. well specs like unit tests and such. Again just suggestion for polishing small things and not something that woudl be required for merge.